### PR TITLE
fix: add support for internal pyxis endpoints

### PR DESCRIPTION
**Pull request by scoheb**
- with the introduction of Konflux internal clusters, we need to interact with internal pyxis endpoints.

### CODE REVIEW

**Review by ralphbean — DISMISSED** [review]

File: tasks/collect-pyxis-params/README.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ for downstream tasks to use.
 |------|-------------|----------|---------------|
 | dataPath | Path to the merged data JSON file generated by collect-data task and containing the pyxis configuration options to use | Yes | data.json |
 
+## Changes in 0.2.1
+* Update the description for `server type` result
```
Comment (johnbieren) [inline tasks/collect-pyxis-params/README.md]: It's just called `server`

**Review by johnbieren — COMMENTED** [review]
I assume you tested the urls? I only ask because I am surprised `api` is not part of them

File: tasks/collect-pyxis-params/README.md (hunk 1 above)
Comment (scoheb) [reply tasks/collect-pyxis-params/README.md]: fixed README

**Comment by scoheb** [comment]
> I assume you tested the urls? I only ask because I am surprised `api` is not part of them

I tested `pyxis.stage.engineering.com`. Therefore `pyxis.engineering.redhat.com` should be correct.

**Review by johnbieren — APPROVED** [review]

**Pull request merged**